Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN mkdir /opt/spark-nb; cd /opt/spark-nb\
   && wget -qO- "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=245467_4d5417147a92418ea8b615e228bb6935" | tar -xvz \
   && wget -qO- "https://archive.apache.org/dist/spark/spark-3.0.3/spark-3.0.3-bin-hadoop3.2.tgz" | tar -xvz \
   && pip install findspark numpy pandas matplotlib sklearn \
-  && wget "https://aerospike.com/artifacts/aerospike-spark/3.2.0/aerospike-spark-assembly-3.2.0.jar"
+  && wget "https://download.aerospike.com/artifacts/aerospike-spark/3.4.0_spark3.1/aerospike-spark-3.4.0_spark_3.1_allshaded.jar"
```

Comment:
Need to use the unshaded jar:

&& wget "https://download.aerospike.com/artifacts/aerospike-spark/3.4.0_spark3.1/aerospike-spark-3.4.0_spark_3.1_clientunshaded.jar"